src/java/davmail/DavGateway.java

```diff
@@ -183,7 +184,16 @@ public static void start() {
         if (showStartupBanner) {
             DavGatewayTray.info(new BundleMessage("LOG_DAVMAIL_GATEWAY_LISTENING", currentVersion, messages));
         }
-        if (!errorMessages.isEmpty()) {
+        if (errorMessages.isEmpty()) {
+            File f = new File("/run/systemd/system");
+            if (f.isDirectory()) {
+                try {
+                    Class.forName("davmail.util.SystemdNotify").getMethod("ready").invoke(null);
+                } catch (Exception e) {
+                    LOGGER.warn("Running on a systemD host but could not notify reaydness.");
```

Comment:
"Readiness" is misspelled here. Please fix.

Also, capitalizing the "d" in "systemd" is a bit unusual. I suggest rephrasing the first part as "Running on a host with systemd" etc.